sdk/react/languageSelectionPreview.js

```diff
@@ -18,6 +18,7 @@ var {
 var LanguageSelectionPreview = React.createClass({
   propTypes: {
     config: React.PropTypes.object,
+    selectedLanguage: React.PropTypes.string,
```

Comment:
will `selectedLanguage` be short form or long form depending on the platform? (ios, android) Or did you already solve the issue of android and ios sending the language in a different format?